packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -1053,17 +1053,14 @@ class Route extends EmberObject implements IRoute {
 
```

Comment:
The verbiage just above here is pretty confusing too:

> If you call `this.transitionTo` from inside of this hook, this route will not be entered in favor of the other hook.

Maybe you could update that also? I'm thinking something like:

```
Calling `this.transitionTo` from inside of the `redirect` hook will cause the current transition (into the route that has implemented `redirect`) to be aborted.
```

Reply:
Done in #18766